### PR TITLE
Pandas Doc Theme

### LINKED ISSUE #158: Pandas Doc Theme
The docs should use the pandas doc theme.

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Closes #158 by applying the pandas doc theme to [the documentation](https://compose.alteryx.com/en/pandas_doc_theme/index.html).

### CODE REVIEW

**Comment by rwedge** [comment]
the "main concepts" page is still searchable but does not appear to be reachable via the "next" button or the nav bar

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+    :hidden:
+    :maxdepth: 1
+
+    install
+    getting_started
+    main_concepts
+    user_guide
+    examples
+    api_reference
+    changelog
```
Comment (rwedge) [inline docs/source/index.rst]: let's rename changelog to "release notes".  In general, let's try to match the EvalML navbar scheme

File: docs/source/user_guide.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+==========
+User Guide
+==========
+
+.. toctree::
+    :glob:
+    :maxdepth: 2
```
Comment (rwedge) [inline docs/source/user_guide.rst:7]: For this page and examples.rst, let's reduce the max depth and add some intro text so the page isn't just a table of contents

**Comment by rwedge** [comment]
* Can we drop sphinx rtd theme as a requirement
* Can we name our other top-level topics to match [evalml'ls](https://evalml.alteryx.com)

File: docs/source/user_guide.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+==========
+User Guide
+==========
+
+In the guides below, a variety of topics are covered so you can learn how to use Compose as productive as possible.
```
Comment (rwedge) [inline docs/source/user_guide.rst:5]: I think this sentence could be more to the point

Comment (jeff-hernandez) [reply docs/source/user_guide.rst:5]: Hmm, what about this?
> In the guides below, you will learn how to use label transforms and more to help you generate training examples more effectively.

File: docs/source/install.rst [hunk 4]
```diff
@@ -2,6 +2,28 @@
 Install
 =======
 
-In Python 3.6 or later, Compose can be installed by running the following commmand:
+Compose is available on PyPI and Conda-forge for Python version 3.6 or later.
+
+:code:`pip`
```
Comment (rwedge) [inline docs/source/install.rst:7]: I find the pink code formatting a little harsh / jarring - can we use the regular heading formatting for the pip and conda sections?

**Comment by rwedge** [comment]
Screen Shot 2020-09-09 at 2 17 58 PM

This warning seems to be specific to the new docs theme

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**